File: src/main/java/me/reckter/telegram/model/File.java
package me.reckter.telegram.model;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @author hannes
 */
public class File extends BaseModel {


    @JsonProperty("file_id")
    private String id;

    @JsonProperty("file_size")
    int size;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public int getSize() {
        return size;
    }

    public void setSize(int size) {
        this.size = size;
    }
}

File: src/main/java/me/reckter/telegram/model/File.kt
package me.reckter.telegram.model

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * @author hannes
 */
open class File : BaseModel() {

    @JsonProperty("file_id")
    var id: String? = null

    @JsonProperty("file_size")
    var size: Int = 0

    @JsonProperty("file_path")
    var path: String? = null
}
